snmp/tests/test_e2e_core_profiles/test_profile_ibm_lenovo_server.py

```diff
@@ -36,7 +36,7 @@ def test_e2e_profile_ibm_lenovo_server(dd_agent_check):
         'device_ip:' + ip_address,
         'device_id:default:' + ip_address,
     ] + [
-        'ibm_imm_machine_level_product_name:Jaded acted their but acted kept driving ' 'Jaded Jaded',
+        'ibm_imm_machine_level_product_name:Jaded acted their but acted kept driving Jaded Jaded',
```

Comment:
@AAraKKe Do you need documentation review of this PR? I don't have any context for this.

Reply:
Thank you  for confirming!